trellis/vault.md

```diff
@@ -86,7 +86,7 @@ Run `trellis vault` to see usage details.
 
 ## Working with vault variables
 
-Here are a few tips for working with [variables and vault](http://docs.ansible.com/ansible/playbooks_best_practices.html#variables-and-vaults) in Trellis.
+Here are a few tips for working with [variables and vault](https://docs.ansible.com/ansible/latest/tips_tricks/ansible_tips_tricks.html#variables-and-vaults) in Trellis.
```

Comment:
🤔 this link didn't work for me either.

Think it's now: https://docs.ansible.com/ansible/latest/tips_tricks/ansible_tips_tricks.html#keep-vaulted-variables-safely-visible